lib/tasks/update_export_health_certificate_paths.rake

```diff
@@ -0,0 +1,153 @@
+    "/export-health-certificates/export-fresh-or-chilled-hides-and-skins-of-ungulates-to-the-european-union-certificate-8327",
+    "/export-health-certificates/export-treated-hides-and-skins-of-ungulates-to-or-transit-through-the-european-union-certificate-8328",
+    "/export-health-certificates/export-treated-hides-and-skins-of-ruminants-and-equidae-that-have-been-separated-for-21-days-to-the-european-union-certificate-8329",
+    "/export-health-certificates/export-of-game-trophies-or-other-preparations-of-birds-and-ungulates-to-the-european-union-certificate-8330",
+    "/export-health-certificates/export-pig-bristles-from-third-countries-free-from-african-swine-fever-to-the-european-union-certificate-8331",
+    "/export-health-certificates/export-fish-oil-not-intended-for-human-consumption-to-or-transit-through-the-european-union-certificate-8332",
+    "/export-health-certificates/export-processed-animal-protein-pap-other-than-types-derived-from-farmed-insects-to-the-european-union-certificate-8333",
+    "/export-health-certificates/movement-back-to-the-european-union-of-registered-horses-for-racing-competition-and-cultural-events-after-temporary-import-to-the-uk-for-a-period-of-less-than-30-days-certificate-8334",
+    "/export-health-certificates/permanently-export-an-equine-animal-for-breeding-or-production-to-the-european-union-certificate-8335",
+    "/export-health-certificates/temporarily-export-a-registered-horse-for-a-period-of-less-than-90-days-to-the-european-union-certificate-8336",
+    "/export-health-certificates/export-a-live-equine-animal-destined-for-another-third-country-which-will-transit-the-european-union-certificate-8337",
+    "/export-health-certificates/export-wild-leporidae-farmed-rabbits-and-wild-land-mammals-other-than-ungulates-for-the-transit-through-or-storage-in-the-european-union-certificate-8339",
+    "/export-health-certificates/export-intermediate-products-to-be-used-for-the-manufacture-of-medicinal-and-cosmetic-products-intended-for-dispatch-to-or-for-transit-through-the-european-union-certificate-8341",
+    "/export-health-certificates/export-bone-horn-and-hoof-products-excluding-meal-intended-for-use-other-than-as-feed-material-organic-fertilisers-or-soil-improvers-for-dispatch-to-or-for-transit-through-the-european-union-certificate-8342",
+    "/export-health-certificates/export-untreated-wool-and-hair-to-the-european-union-certificate-8343",
+  ]
+  content_ids = []
+  redirect_path = "/starting-to-export/licences"
+
+  paths.each do |path|
+    unpublishing = Edition.live.find_by(base_path: path).unpublishing
+    redirect_details = [{
+                      "path"        =>  path,
+                      "type"        =>  "exact",
+                      "destination" =>  redirect_path,
+
+                    }]
+    unpublishing.redirects = redirect_details
+    unpublishing.type = "redirect"
+    unpublishing.save!
```

Comment:
It's pretty irregular to work with the data in the Publishing API directly, rather than going through the API. Is there an option of unpublishing this content from a publishing app?

Reply:
As I understand it, Specialist Publisher doesn't have it's own database but also reads from the publishing-api, so having discussed with @thomasleese this was the suggested approach